Marlin/src/feature/bedlevel/bdl/bdl.cpp

```diff
@@ -149,8 +148,10 @@ void BDS_Leveling::process() {
         }
       #endif
     }
-    else
-      stepper.apply_directions();
```

Comment:
I'm taking the chance that whatever issue existed before that required `stepper.apply_directions()` is now fixed. We should check with and without `INTEGRATED_BABYSTEPPING`. I might actually remove the `INTEGRATED_BABYSTEPPING` option and make integrated babystepping the only kind of babystepping since it has been pretty well tested and generally works better.

Reply:
Thanks, I will also test.

Reply:
> I might actually remove the INTEGRATED_BABYSTEPPING option and make integrated babystepping the only kind of babystepping since it has been pretty well tested and generally works better.

+1 for this idea. I enable it on every build I maintain now and would consider it "ready for production".